packages/flame/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/gestures.dart';
+
+import '../../extensions.dart';
+
+abstract class BaseInfo<T> {
```

Comment:
I like having some inheritance, how about BaseEvent with just the raw event and then a SinglePositionEvent that adds the position? that would help people to define generic event handlers for multiple events